### ISSUE TITLE
No option to dismiss/turn off update notifications

### ISSUE THREAD

**Issue by bagley** [streetwriters/notesnook#2775]
### What happened?

TL;DR: Please add option to dismiss or turn off update notifications, so we can update when convenient.

I've had a very frustrating first experience with this app because of this annoying "update notification" popup. It pops up each and every time I open the app. I have 2.5.2 installed from F-droid, and 2.5.3 just came out and hasn't reached F-droid yet. But still, it's been out for only 6 hours (as of this writing), yet I've seen that update notification a dozen times. That seems excessive.

Potential scenarios where the user could get constant notifications.
- If the update breaks the app and user must use an previous version until the next update (or two) corrects it.
- If there's an issue with the update function in the app, and it thinks it's out of date.
- If there's a holdup in Play / F-droid / whereever the user got it from.
- Back to back security bugs that had to be pushed out, where users are having to update every day, if not every few hours.

Sure, one can just install from Github, but they may not want to. Besides, this constant notifying feels forced. That one must update right this very moment. Your user base is likely people who are very active and productive, so this may just get in the way of their day and become very annoying.

Ideas:
- Only prompt the user _once_ every 2 weeks about the update. (at most once every 3 days).
- Allow them to even disable it, and instead put a notice at the top of the settings page.
- Have a separate update check url if the app was installed from Play/F-droid, to properly let them know and install from the same place they originally got it from. (I installed from F-droid, so waiting for the update to post there).

Disabling the app's access to the network is a workaround, but then sync obviously won't work.

Related:
- #2265 (posted by the Streetwriters bot).
- #2774

### Steps to reproduce the problem

Steps
- open the app that's not updated.
- consistently be told about an update
- apk on Github not signed.
- no option to turn off update checks to wait for it to reach F-droid/Play/etc.

### Version

2.5.2

### Platform/OS

Android

**Comment by danielintempesta** [comment]
Same problem.

>     * Allow them to even disable it, and instead put a notice at the top of the settings page.

This is the best option.

**Comment by thecodrr** [comment]
We will work on improving the flow around updates in the coming releases.

**Comment by mockinbirdy** [comment]
I second that. Usually new updates are buggy so lots of users don't want to mess things up updating the app every couple of days.
